tests/integration/037_vcs_connection/main.tf

```diff
@@ -0,0 +1,16 @@
+provider "random" {}
+
+resource "random_string" "random" {
+  length    = 8
+  special   = false
+  min_lower = 8
+}
+
+# resource "env0_vcs_connection" "github" {
```

Comment:
I can't run this test because I'm unsure if an agent (or what) is configured.
I will require assistance with this specific test. I am open to suggestions.